### PR TITLE
Test patch

### PR DESCRIPTION
**Pull request by MateoLostanlen**
Following the discussions in #5 I propose you the following solution inspired by this repo : https://pypi.org/project/fake-rpi/
We create fakes Raspberry pi library classes for the tests, and we use the pi_patch module to replace the real ones by the fake ones during the tests. I find this solution quite elegant and it solves our problem. It's not perfect because we don't reproduce the exact working conditions but it's similar

What do you think ?

### CODE REVIEW

**Comment by MateoLostanlen** [comment]
The broken link is the one of the docs and flake8 can't be done better in pi_patch module

File: test/pi_patch.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the GNU Affero General Public License version 3.
+# See LICENSE or go to <https://www.gnu.org/licenses/agpl-3.0.txt> for full license details.
+
+import sys
+import fake_gpiozero
```
Comment (fe51) [inline test/pi_patch.py:7]: Does `fake_gpiozero` should be added to the dev requirements ?

**Review by fe51 — COMMENTED** [review]
Thanks for the PR ! 

I am not so much knowledgeable, but its seems a nice way to tests code using ARM elts !

Except requirements question,  looks okay to me :)

File: test/pi_patch.py (hunk 1 above)
Comment (MateoLostanlen) [reply test/pi_patch.py:7]: no we do not whant to use it in the real package, it's just for testing on github

Comment (MateoLostanlen) [reply test/pi_patch.py:7]: moreover it's not a package so can't be installed ;)

Comment (fe51) [reply test/pi_patch.py:7]: ho okay! So how to install it ?

**Review by fe51 — APPROVED** [review]

File: test/pi_patch.py (hunk 1 above)
Comment (fe51) [reply test/pi_patch.py:7]: okay my bad I just get the tricks ^^Nice

**Pull request merged**